packages/block/src/block.ts

```diff
@@ -107,17 +110,19 @@ export class Block {
 
     // parse uncle headers
     const uncleHeaders = []
+    const uncleOpts: any = {
```

Comment:
(same question here if you tried to type this as `BlockOptions`, could give us some better security when working on the opts in the future)

Reply:
This code improvement works but is very messy and likely suboptimal, I was already pretty sleepy. 🙂

Would be great if someone could improve this at done point.

Reply:
I'm working on a smallish Type-improvements PR, I'll see if I can easily fit that in!

Reply:
Cool. 🙂

Reply:
Implemented in #1480